build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 
   id 'com.github.ben-manes.versions' version '0.28.0'
 
-  id 'com.diffplug.gradle.spotless' version '4.5.0' apply false
+  id 'com.diffplug.gradle.spotless' version '4.3.0' apply false
```

Comment:
Why is that?

Reply:
Idk why exactly, but 4.5.0 sometimes seemed to ignore un-formatted files (both in `apply` and in `check`)

Reply:
Possibly the combination of Gradle version (6.4) and plugin version was responsible? dunno